app/Http/Repositories/MailChimpRepository.php

```diff
@@ -302,7 +335,22 @@ public function subscribeMember(string $listId, string $email, array $interests,
         return $response;
     }
 
-    private function toDtoArray(array $resultSet) : array
+    private function toSegmentDtoArray(array $resultSet) : array
```

Comment:
I'm sure you already know this, but since we renamed this function, we should double check and update any instances of this function.

Reply:
@kassanj Yep! Thanks fro the reminder. I just double-checked and looks like my IDE did that for me. I see in the diff here that the couple of methods that were using it have def been udpated.